src/msw/textctrl.cpp

```diff
@@ -1114,6 +1114,115 @@ void wxTextCtrl::DoSetValue(const wxString& value, int flags)
     }
 }
 
+#if wxUSE_RICHEDIT
+
+DWORD wxCALLBACK MSWEditStreamOutCallback(DWORD_PTR dwCookie, LPBYTE pbBuff,
+    LONG cb, LONG* WXUNUSED(pcb))
+{
+#if wxUSE_THREADS
+    wxMutexGuiLeaveOrEnter();
+#endif // wxUSE_THREADS
+    MSG msg;
+    while ( ::PeekMessage(&msg, (HWND)0, 0, 0, PM_REMOVE) )
+    {
+        if (msg.message != WM_QUIT)
+        {
+            ::TranslateMessage(&msg);
+            ::DispatchMessage(&msg);
+        }
+    }
+
```

Comment:
Sorry, why do we need to do this here? This definitely needs a comment but I really hope we can avoid any message dispatching as this is very dangerous, nobody expects any callbacks to be invoked when calling `wxTextCtrl::GetRTFValue()`!

Also, we never did this while we had `wxRichEditStreamOut` (removed in 4519d8e08a (Remove wxUSE_UNICODE checks as they're always true now, 2022-10-27), see its parent commit for its last version), so I really hope it's just a leftover and can be removed entirely.

Reply:
Yes, this was from Windows XP days when streaming into this control was substantially slower. I'll remove this block.